modules/spellchecker/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "spellchecker",
+  "fullName": "Spell Checker",
+  "version": "1.0.0",
+  "status": "experimental",
+  "description": "An easy to use spell checker for all botpress",
+  "private": true,
+  "main": "dist/backend/index.js",
+  "scripts": {
+    "build": "node ../../build/module-builder/bin/entry build",
+    "watch": "node ../../build/module-builder/bin/entry watch",
+    "package": "node ../../build/module-builder/bin/entry package"
+  },
+  "author": "Botpress, Inc.",
+  "license": "AGPL-3.0-only",
+  "dependencies": {
+    "axios": "^0.21.1",
+    "nodehun": "^3.0.2",
```

Comment:
This library comes with a node N-API extension, which should have been pre-compiled and provided with Botpress